runtime-modules/content/src/permissions/mod.rs

```diff
@@ -152,7 +158,7 @@ pub fn ensure_actor_authorized_to_delete_channel<T: Trait>(
             ensure_member_is_channel_owner::<T>(channel_owner, member_id)?;
             Ok(())
         }
-        // Lead & collaborators should use their member or curator role in order to update reward account.
+        // collaborators should use their member or curator role in order to update reward account.
```

Comment:
The comment is about reward account, but this is `ensure_actor_authorized_to_delete_channel` function